helmfiles/subhelmfiles/jenkins.helmfile.yaml

```diff
@@ -66,11 +77,38 @@ releases:
               welcome-message: |
                 jenkins:
                   systemMessage: "Ephemeral Virtual Training Range (EVTR)"
+              credentials: |
+                credentials:
+                  system:
+                    domainCredentials:
+                      - credentials:
+                          - gitLabApiTokenImpl:
+                              scope: SYSTEM
+                              id: gitlab_token
+                              apiToken: "{{$gitlabApiToken}}"
+                              description: "Gitlab Token"
+              gitlab: |
+                unclassified:
+                  gitlabconnectionconfig:
+                    connections:
+                      - apiTokenId: gitlab_token
+                        clientBuilderId: "autodetect"
+                        connectionTimeout: 20
+                        ignoreCertificateErrors: true
+                        name: "{{$gitlabHostName}}"
+                        readTimeout: 10
+                        url: "https://{{$gitlabHostName}}"
             securityRealm: |-
-              legacy
+              gitLab:
+                clientID: "{{$jenkinsId}}"
+                clientSecret: "{{$jenkinsSecret}}"
+                gitlabApiUri: "https://{{$gitlabHostName}}"
+                gitlabWebUri: "https://{{$gitlabHostName}}"
             authorizationStrategy: |-
-              loggedInUsersCanDoAnything:
-                allowAnonymousRead: false
+              projectMatrix:
+                permissions:
+                - "Overall/Administer:authenticated"
```

Comment:
Students shouldn't receive admin access. How much more work would it be to fix that? If it is quick, let's do it now. If not, let's add a story to do it to the backlog.

Reply:
I've reconfigured to only the "root" user in Gitlab has Administer permissions. Authenticated users are read only.

Reply:
👍